### PR TITLE
feat: Checkbox and @tailwind/forms

### PR DESCRIPTION
**Pull request by wit03**
Done finally

### CODE REVIEW

**Comment by riflowth** [comment]
@wit03 Why should we need to use `@tailwindcss/forms`?

**Comment by wit03** [comment]
It is quite an edge case for the checkbox and radio, we can't natively use only tailwind to style them. So that's the reason.

**Comment by riflowth** [comment]
We can use pure css to avoid dependency but it's ok.

BTW, you need to use your component to create this page
image

**Comment by riflowth** [comment]
If you too busy we can do this task for you @wit03

**Comment by wit03** [comment]
Ok, seems like the requirements in the issues (referring to #13) are not the same, might be your or my confusion. But I recommend to merge and create new backlog that given more details about this.

> We can use pure css to avoid dependency but it's ok.
> 
> BTW, you need to use your component to create this page image

**Comment by riflowth** [comment]
I will do the task all of this myself. You don't have to do anything. Thank you for work 👍  @wit03

**Review by riflowth — APPROVED** [review]
Seems OK

**Comment by riflowth** [comment]
> Seems OK

Arhhh! Your component cannot be used, but it's ok. I hardcoded in f1ca7cdd9f395a4809fdfde321c372b3e9b55a9a

File: apps/registration/src/components/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    setObj,
+    name,
+}: {
+    obj: boolean,
+    setObj: any,
+    name?: string
+
+}) => {
+    const handleChange = () => {
+        setObj(!obj)
```
Comment (wit03) [inline apps/registration/src/components/Checkbox.tsx:13]: Geez, seems like I it shouldn't work btw. Sry for messy code

**Pull request merged**